lib/prompts/accountsPrompt.ts

```diff
@@ -28,23 +27,6 @@ const selectAccountFromConfig = async (config, prompt) => {
   return selectedDefault;
 };
 
-const selectAndSetAsDefaultAccountPrompt = async config => {
```

Comment:
This prompt does exactly the same thing as the one above it. I added a call to `updateDefaultAccount()` in the two places that this was being used